### ISSUE TITLE
New submissions for Mon, 20 Sep 21

### ISSUE THREAD

**Issue by LeeKyungwook** [LeeKyungwook/get-arxiv-noti#116]
### A computationally efficient framework for vector representation of  persistence diagrams
 - **Authors:** Authors: Kit C. Chan, Umar Islambekov, Alexey Luchinsky, Rebecca Sanders
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Computational Geometry (cs.CG)
 - **Arxiv link:** https://arxiv.org/abs/2109.08239
 - **Pdf link:** https://arxiv.org/pdf/2109.08239
 - **Abstract**
 In Topological Data Analysis, a common way of quantifying the shape of data is to use a persistence diagram (PD). PDs are multisets of points in $\mathbb{R}^2$ computed using tools of algebraic topology. However, this multi-set structure limits the utility of PDs in applications. Therefore, in recent years efforts have been directed towards extracting informative and efficient summaries from PDs to broaden the scope of their use for machine learning tasks. We propose a computationally efficient framework to convert a PD into a vector in $\mathbb{R}^n$, called a vectorized persistence block (VPB). We show that our representation possesses many of the desired properties of vector-based summaries such as stability with respect to input noise, low computational cost and flexibility. Through simulation studies, we demonstrate the effectiveness of VPBs in terms of performance and computational cost within various learning tasks, namely clustering, classification and change point detection.
### Towards agricultural autonomy: crop row detection under varying field  conditions using deep learning
 - **Authors:** Authors: Rajitha de Silva, Grzegorz Cielniak, Junfeng Gao
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Robotics (cs.RO)
 - **Arxiv link:** https://arxiv.org/abs/2109.08247
 - **Pdf link:** https://arxiv.org/pdf/2109.08247
 - **Abstract**
 This paper presents a novel metric to evaluate the robustness of deep learning based semantic segmentation approaches for crop row detection under different field conditions encountered by a field robot. A dataset with ten main categories encountered under various field conditions was used for testing. The effect on these conditions on the angular accuracy of crop row detection was compared. A deep convolutional encoder decoder network is implemented to predict crop row masks using RGB input images. The predicted mask is then sent to a post processing algorithm to extract the crop rows. The deep learning model was found to be robust against shadows and growth stages of the crop while the performance was reduced under direct sunlight, increasing weed density, tramlines and discontinuities in crop rows when evaluated with the novel metric.
### The Stackelberg Equilibrium for One-sided Zero-sum Partially Observable  Stochastic Games
 - **Authors:** Authors: Wei Zheng, Taeho Jung, Hai Lin
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2109.08339
 - **Pdf link:** https://arxiv.org/pdf/2109.08339
 - **Abstract**
 Formulating cyber-security problems with attackers and defenders as a partially observable stochastic game has become a trend recently. Among them, the one-sided two-player zero-sum partially observable stochastic game (OTZ-POSG) has emerged as a popular model because it allows players to compete for multiple stages based on partial knowledge of the system. All existing work on OTZ-POSG has focused on the simultaneous move scenario and assumed that one player's actions are private in the execution process. However, this assumption may become questionable since one player's action may be detected by the opponent through deploying action detection strategies. Hence, in this paper, we propose a turn-based OTZ-POSG with the assumption of public actions and investigate the existence and properties of a Stackelberg equilibrium for this game. We first prove the existence of the Stackelberg equilibrium for the one-stage case and show that the one-stage game can be converted into a linear-fractional programming problem and therefore solved by linear programming. For multiple stages, the main challenge is the information leakage issue as the public run-time action reveals certain private information to the opponent and allows the opponent to achieve more rewards in the future. To deal with this issue, we adopt the concept of $\epsilon$-Stackelberg equilibrium and prove that this equilibrium can be achieved for finite-horizon OTZ-POSGs. We propose a space partition approach to solve the game iteratively and show that the value function of the leader is piece-wise linear and the value function of the follower is piece-wise constant for multiple stages. Finally, examples are given to illustrate the space partition approach and show that value functions are piece-wise linear and piece-wise constant.
### LOF: Structure-Aware Line Tracking based on Optical Flow
 - **Authors:** Authors: Meixiang Quan, Zheng Chai, Xiao Liu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Robotics (cs.RO)
 - **Arxiv link:** https://arxiv.org/abs/2109.08466
 - **Pdf link:** https://arxiv.org/pdf/2109.08466
 - **Abstract**
 Lines provide the significantly richer geometric structural information about the environment than points, so lines are widely used in recent Visual Odometry (VO) works. Since VO with lines use line tracking results to locate and map, line tracking is a crucial component in VO. Although the state-of-the-art line tracking methods have made great progress, they are still heavily dependent on line detection or the predicted line segments. In order to relieve the dependencies described above to track line segments completely, accurately, and robustly at higher computational efficiency, we propose a structure-aware Line tracking algorithm based entirely on Optical Flow (LOF). Firstly, we propose a gradient-based strategy to sample pixels on lines that are suitable for line optical flow calculation. Then, in order to align the lines by fully using the structural relationship between the sampled points on it and effectively removing the influence of sampled points on it occluded by other objects, we propose a two-step structure-aware line segment alignment method. Furthermore, we propose a line refinement method to refine the orientation, position, and endpoints of the aligned line segments. Extensive experimental results demonstrate that the proposed LOF outperforms the state-of-the-art performance in line tracking accuracy, robustness, and efficiency, which also improves the location accuracy and robustness of VO system with lines.
### Including Keyword Position in Image-based Models for Act Segmentation of  Historical Registers
 - **Authors:** Authors: Mélodie Boillet, Martin Maarand, Thierry Paquet, Christopher Kermorvant
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2109.08477
 - **Pdf link:** https://arxiv.org/pdf/2109.08477
 - **Abstract**
 The segmentation of complex images into semantic regions has seen a growing interest these last years with the advent of Deep Learning. Until recently, most existing methods for Historical Document Analysis focused on the visual appearance of documents, ignoring the rich information that textual content can offer. However, the segmentation of complex documents into semantic regions is sometimes impossible relying only on visual features and recent models embed both visual and textual information. In this paper, we focus on the use of both visual and textual information for segmenting historical registers into structured and meaningful units such as acts. An act is a text recording containing valuable knowledge such as demographic information (baptism, marriage or death) or royal decisions (donation or pardon). We propose a simple pipeline to enrich document images with the position of text lines containing key-phrases and show that running a standard image-based layout analysis system on these images can lead to significant gains. Our experiments show that the detection of acts increases from 38 % of mAP to 74 % when adding textual information, in real use-case conditions where text lines positions and content are extracted with an automatic recognition system.
### An open GPS trajectory dataset and benchmark for travel mode detection
 - **Authors:** Authors: Jinyu Chen, Haoran Zhang, Xuan Song, Ryosuke Shibasaki
 - **Subjects:** Machine Learning (cs.LG); Databases (cs.DB)
 - **Arxiv link:** https://arxiv.org/abs/2109.08527
 - **Pdf link:** https://arxiv.org/pdf/2109.08527
 - **Abstract**
 Travel mode detection has been a hot topic in the field of GPS trajectory-related processing. Former scholars have developed many mathematical methods to improve the accuracy of detection. Among these studies, almost all of the methods require ground truth dataset for training. A large amount of the studies choose to collect the GPS trajectory dataset for training by their customized ways. Currently, there is no open GPS dataset marked with travel mode. If there exists one, it will not only save a lot of efforts in model developing, but also help compare the performance of models. In this study, we propose and open GPS trajectory dataset marked with travel mode and benchmark for the travel mode detection. The dataset is collected by 7 independent volunteers in Japan and covers the time period of a complete month. The travel mode ranges from walking to railway. A part of routines are traveled repeatedly in different time slots to experience different road and travel conditions. We also provide a case study to distinguish the walking and bike trips in a massive GPS trajectory dataset.
### Exploring Multitask Learning for Low-Resource AbstractiveSummarization
 - **Authors:** Authors: Ahmed Magooda, Mohamed Elaraby, Diane Litman
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2109.08565
 - **Pdf link:** https://arxiv.org/pdf/2109.08565
 - **Abstract**
 This paper explores the effect of using multitask learning for abstractive summarization in the context of small training corpora. In particular, we incorporate four different tasks (extractive summarization, language modeling, concept detection, and paraphrase detection) both individually and in combination, with the goal of enhancing the target task of abstractive summarization via multitask learning. We show that for many task combinations, a model trained in a multitask setting outperforms a model trained only for abstractive summarization, with no additional summarization data introduced. Additionally, we do a comprehensive search and find that certain tasks (e.g. paraphrase detection) consistently benefit abstractive summarization, not only when combined with other tasks but also when using different architectures and training corpora.
### Does Commonsense help in detecting Sarcasm?
 - **Authors:** Authors: Somnath Basu Roy Chowdhury, Snigdha Chaturvedi
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2109.08588
 - **Pdf link:** https://arxiv.org/pdf/2109.08588
 - **Abstract**
 Sarcasm detection is important for several NLP tasks such as sentiment identification in product reviews, user feedback, and online forums. It is a challenging task requiring a deep understanding of language, context, and world knowledge. In this paper, we investigate whether incorporating commonsense knowledge helps in sarcasm detection. For this, we incorporate commonsense knowledge into the prediction process using a graph convolution network with pre-trained language model embeddings as input. Our experiments with three sarcasm detection datasets indicate that the approach does not outperform the baseline model. We perform an exhaustive set of experiments to analyze where commonsense support adds value and where it hurts classification. Our implementation is publicly available at: https://github.com/brcsomnath/commonsense-sarcasm.
### The futility of STILTs for the classification of lexical borrowings in  Spanish
 - **Authors:** Authors: Javier de la Rosa
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2109.08607
 - **Pdf link:** https://arxiv.org/pdf/2109.08607
 - **Abstract**
 The first edition of the IberLEF 2021 shared task on automatic detection of borrowings (ADoBo) focused on detecting lexical borrowings that appeared in the Spanish press and that have recently been imported into the Spanish language. In this work, we tested supplementary training on intermediate labeled-data tasks (STILTs) from part of speech (POS), named entity recognition (NER), code-switching, and language identification approaches to the classification of borrowings at the token level using existing pre-trained transformer-based language models. Our extensive experimental results suggest that STILTs do not provide any improvement over direct fine-tuning of multilingual models. However, multilingual models trained on small subsets of languages perform reasonably better than multilingual BERT but not as good as multilingual RoBERTa for the given dataset.
### Monitoring Indoor Activity of Daily Living Using Thermal Imaging: A Case  Study
 - **Authors:** Authors: Hassan M. Ahmed, Bessam Abdulrazak (AMI-Lab Faculte des sciences, Universite de Sherbrooke)
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2109.08672
 - **Pdf link:** https://arxiv.org/pdf/2109.08672
 - **Abstract**
 Monitoring indoor activities of daily living (ADLs) of a person is neither an easy nor an accurate process. It is subjected to dependency on sensor type, power supply stability, and connectivity stability without mentioning artifacts introduced by the person himself. Multiple challenges have to be overcome in this field, such as; monitoring the precise spatial location of the person, and estimating vital signs like an individuals average temperature. Privacy is another domain of the problem to be thought of with care. Identifying the persons posture without a camera is another challenge. Posture identification assists in the persons fall detection. Thermal imaging could be a proper solution for most of the mentioned challenges. It provides monitoring both the persons average temperature and spatial location while maintaining privacy. In this research, we propose an IoT system for monitoring an indoor ADL using thermal sensor array (TSA). Three classes of ADLs are introduced, which are daily activity, sleeping activity and no-activity respectively. Estimating person average temperature using TSAs is introduced as well in this paper. Results have shown that the three activity classes can be identified as well as the persons average temperature during day and night. The persons spatial location can be determined while his/her privacy is maintained as well.
## Keyword: face recognition
There is no result 
### KATANA: Simple Post-Training Robustness Using Test Time Augmentations
 - **Authors:** Authors: Gilad Cohen, Raja Giryes
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2109.08191
 - **Pdf link:** https://arxiv.org/pdf/2109.08191
 - **Abstract**
 Although Deep Neural Networks (DNNs) achieve excellent performance on many real-world tasks, they are highly vulnerable to adversarial attacks. A leading defense against such attacks is adversarial training, a technique in which a DNN is trained to be robust to adversarial attacks by introducing adversarial noise to its input. This procedure is effective but must be done during the training phase. In this work, we propose a new simple and easy-to-use technique, KATANA, for robustifying an existing pretrained DNN without modifying its weights. For every image, we generate N randomized Test Time Augmentations (TTAs) by applying diverse color, blur, noise, and geometric transforms. Next, we utilize the DNN's logits output to train a simple random forest classifier to predict the real class label. Our strategy achieves state-of-the-art adversarial robustness on diverse attacks with minimal compromise on the natural images' classification. We test KATANA also against two adaptive white-box attacks and it shows excellent results when combined with adversarial training. Code is available in https://github.com/giladcohen/KATANA.
### Simple Entity-Centric Questions Challenge Dense Retrievers
 - **Authors:** Authors: Christopher Sciavolino, Zexuan Zhong, Jinhyuk Lee, Danqi Chen
 - **Subjects:** Computation and Language (cs.CL); Information Retrieval (cs.IR)
 - **Arxiv link:** https://arxiv.org/abs/2109.08535
 - **Pdf link:** https://arxiv.org/pdf/2109.08535
 - **Abstract**
 Open-domain question answering has exploded in popularity recently due to the success of dense retrieval models, which have surpassed sparse models using only a few supervised training examples. However, in this paper, we demonstrate current dense models are not yet the holy grail of retrieval. We first construct EntityQuestions, a set of simple, entity-rich questions based on facts from Wikidata (e.g., "Where was Arve Furset born?"), and observe that dense retrievers drastically underperform sparse methods. We investigate this issue and uncover that dense retrievers can only generalize to common entities unless the question pattern is explicitly observed during training. We discuss two simple solutions towards addressing this critical problem. First, we demonstrate that data augmentation is unable to fix the generalization problem. Second, we argue a more robust passage encoder helps facilitate better question adaptation using specialized question encoders. We hope our work can shed light on the challenges in creating a robust, universal dense retriever that works well across different input distributions.
### Mitigating Data Scarceness through Data Synthesis, Augmentation and  Curriculum for Abstractive Summarization
 - **Authors:** Authors: Ahmed Magooda, Diane Litman
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2109.08569
 - **Pdf link:** https://arxiv.org/pdf/2109.08569
 - **Abstract**
 This paper explores three simple data manipulation techniques (synthesis, augmentation, curriculum) for improving abstractive summarization models without the need for any additional data. We introduce a method of data synthesis with paraphrasing, a data augmentation technique with sample mixing, and curriculum learning with two new difficulty metrics based on specificity and abstractiveness. We conduct experiments to show that these three techniques can help improve abstractive summarization across two summarization models and two different small datasets. Furthermore, we show that these techniques can improve performance when applied in isolation and when combined.
